diff -ruN rogue-5.3dupa/instruct.c rogue-5.3/instruct.c
--- rogue-5.3dupa/instruct.c	Sat Feb  9 17:23:56 2002
+++ rogue-5.3/instruct.c	Sat Feb  9 17:26:58 2002
@@ -9,7 +9,7 @@
 #include <string.h>
 #include <stdio.h>
 
-static char *instructions="/usr/games/rogue.instr";
+static char *instructions="/var/games/rogue/rogue.instr";
 
 void Instructions()
 {
diff -ruN rogue-5.3dupa/score.c rogue-5.3/score.c
--- rogue-5.3dupa/score.c	Sat Feb  9 17:23:57 2002
+++ rogue-5.3/score.c	Sat Feb  9 17:27:08 2002
@@ -16,7 +16,7 @@
 #include <stdio.h>
 #include "rogue.h"
 
-char *score_file = "/usr/games/rogue.scores";
+char *score_file = "/var/games/rogue/rogue.scores";
 
 extern char login_name[];
 extern char *m_names[];
